test/test.js

```diff
@@ -75,6 +75,11 @@ exports['After 30 years a borrower with a 30 year, $180,000 loan with a 4.25% in
   test.done();
 };
 
+exports['After 30 years a borrower with a 30 year, $180,000 loan with a 4.25% interest rate paying $200 extra per month will start paying more in principal after 56 months'] = function (test) {
```

Comment:
Could you add another test case or two and link to an existing online calculator that demonstrates the same results in a comment on this PR?

Reply:
There aren't any calculators that do this, hence why I added it.  With [bank rate](https://www.bankrate.com/calculators/mortgages/mortgage-calculator.aspx) you can enter extra principal payments and look at the graph then figure it out.